hack/test-e2e.sh

```diff
@@ -4,6 +4,8 @@ set -e -x -u
 
 go clean -testcache
 
+# create ns if not exists because the `apply -f -` won't complain if fed an empty yaml to apply due to the ns already existing.
```

Comment:
i dont think `kubectl create` will ever return empty yaml

Reply:
that's fair , probably more of a "no-op" yaml. i'll adjust the comment.